### DISCUSSION TITLE
Read later button

### DISCUSSION THREAD

**Discussion by guionardo** [dailydotdev/daily#528] [Feature Requests]
How about a read later button on the cards?
Every time I found a interesting link to read and don't have time to, I save the link to my favourites browser. This is because sometimes, in the next day, the link is gone.
In a week, I have 50 links to read. 

This feature could add a new page near Bookmarks and Reading history with a badge counting how many links I want to read later.

**Comment by idoshamun** [comment]
Hi @guionardo,

Thanks for the suggestion, it's a great idea and we already have a discussion going on here:
https://github.com/dailydotdev/daily/discussions/364

I'm locking the conversation here so we can centralize the discussion
